client/my-sites/site-indicator/style.scss

```diff
@@ -111,11 +111,16 @@
 
 	// Links within the action message
 	a,
+	a:visited,
 	.button.is-borderless.is-scary,
 	.site-indicator__action-button {
 		border-bottom: 1px solid rgba( 255, 255, 255, 0.6 );
 		border-radius: 0;
 		color: $white;
 		text-decoration: none;
 	}
+
+	a > .gridicons-external {
```

Comment:
May not need changes here if we use the external icon component

Reply:
Some custom styles are still there, as `ExternalLink` doesn't look top notch with small (12px) font sizes. For example, the margins around the icon are better expressed with `em` unitls rather than absolute `px`. Writing down a todo to tweak the `ExternalLink` itself.